mkdocs.yml

```diff
@@ -125,6 +125,7 @@ nav:
   - 'Solo Proof-of-Stake (PoS) Voting': 'advanced/solo-proof-of-stake-voting.md'
   - 'Operating a VSP': 'advanced/operating-a-vsp.md'
   - 'Security': 'advanced/general-security.md'
+  - 'Atomic Swap': 'advanced/atomic-swap.md'
```

Comment:
Please change the menu entry to "Atomic Swaps" matching the heading in the article (suffix "s")